lib/govuk_test/railtie.rb

```diff
@@ -0,0 +1,9 @@
+require "govuk_test"
+
+module GovukTest
+  class Railtie < Rails::Railtie
+    rake_tasks do
+      Dir[File.expand_path('../tasks/*.rake', File.dirname(__FILE__))].each { |f| load f }
```

Comment:
Minor: if we're using a generalised approach for including rake tasks it may be better to use something like `../tasks/**/*.rake` to make sure we include all rake tasks, even if some are created within a subdirectory.

Reply:
Thanks, that's a good idea.